### ISSUE TITLE
Feature request: output to console

### ISSUE THREAD

**Issue by DaVince** [spheredev/neosphere#27]
This would be useful to have because it makes it easy to copy-paste the error, as well as seeing it after closing the game engine.

**Comment by fatcerberus** [comment]
The console is kind of an afterthought in minisphere because I have it hidden for Windows release builds, on which I do most of my testing (debug builds are too slow).  There's actually a Print() function you can call to output stuff to stdout in-game.

That said, it would certainly be trivial to add a printf() in the error catcher.  Thanks for the tip!

**Comment by DaVince** [comment]
Well, I figure that Windows users have a way to run it through the command prompt anyway, so it's not like it won't be beneficial to them when they need to debug. :)

**Comment by fatcerberus** [comment]
I just added a Windows console build to the MSVC project which generates console.exe and console64.exe.  It's a fully optimized release build like the normal Windows build, with the only difference being that it opens a console window.

I'll probably include it in future distributions.

I added this after discovering that the engine doesn't output anything to the console when compiled for release, even if you run it from the command line.  It just immediately returns to the command prompt...
